program/src/processor.rs

```diff
@@ -1049,20 +982,31 @@ pub fn process(program_id: &Pubkey, accounts: &[AccountInfo], input: &[u8]) -> P
         }
         LidoInstruction::Unstake { amount } => process_unstake(program_id, amount, accounts),
         LidoInstruction::UpdateExchangeRate => process_update_exchange_rate(program_id, accounts),
-        LidoInstruction::WithdrawInactiveStake => {
+        LidoInstruction::WithdrawInactiveStake
+        | LidoInstruction::CollectValidatorFee
+        | LidoInstruction::ClaimValidatorFee
+        | LidoInstruction::AddValidator => {
+            msg!("Please use a new instruction {:?}", instruction);
```

Comment:
```suggestion
            msg!("{:?} is no longer supported since v2. Please check the changelog in the repository for update instructions.", instruction);
```

We should also list those breaking changes in `CHANGELOG.md` so we don’t forget to do it later. We can add a preliminary section there, something like

```markdown
## v2.0.0 (unreleased)

New features:

* Solido no longer requires that validators use a 100%-commission account of which Solido
  is the withdraw authority. Any vote account can now be used, as long as its commission does
  not exceed Solido’s configured maximum commission percentage.

**Compatibility**

* The `AddValidator` instruction is no longer supported and has been superseded by `AddValidatorV2`.
* ...
```

Reply:
fixed